diff --git a/node_modules/react-native-web/dist/exports/Image/index.js b/node_modules/react-native-web/dist/exports/Image/index.js
index 5c25ebf..1ba9849 100644
--- a/node_modules/react-native-web/dist/exports/Image/index.js
+++ b/node_modules/react-native-web/dist/exports/Image/index.js
@@ -320,7 +320,7 @@ var Image = /*#__PURE__*/React.forwardRef(function (props, ref) {
   }), hiddenImage, createTintColorSVG(tintColor, filterRef.current));
 });
 Image.displayName = 'Image'; // $FlowIgnore: This is the correct type, but casting makes it unhappy since the variables aren't defined yet
-
+Image.propTypes = { style: () => {}, source: () => {}};
 var ImageWithStatics = Image;
 
 ImageWithStatics.getSize = function (uri, success, failure) {
diff --git a/node_modules/react-native-web/dist/exports/Text/index.js b/node_modules/react-native-web/dist/exports/Text/index.js
index ec43b68..f55b304 100644
--- a/node_modules/react-native-web/dist/exports/Text/index.js
+++ b/node_modules/react-native-web/dist/exports/Text/index.js
@@ -138,6 +138,7 @@ var Text = /*#__PURE__*/React.forwardRef(function (props, forwardedRef) {
   }, element);
 });
 Text.displayName = 'Text';
+Text.propTypes = { style: () => {} };
 var classes = css.create({
   text: {
     border: '0 solid black',
diff --git a/node_modules/react-native-web/dist/index.js b/node_modules/react-native-web/dist/index.js
index 592354f..64c09e2 100644
--- a/node_modules/react-native-web/dist/index.js
+++ b/node_modules/react-native-web/dist/index.js
@@ -1,3 +1,4 @@
+export const ViewPropTypes = { style: () => {} };
 export { default as unstable_createElement } from './exports/createElement';
 export { default as findNodeHandle } from './exports/findNodeHandle';
 export { default as processColor } from './exports/processColor';
